src/core/LfoController.cpp

```diff
@@ -101,9 +101,9 @@ void LfoController::updateValueBuffer()
 	}
 
 	float amount = m_amountModel.value();
-	ValueBuffer *amountBuffer = m_amountModel.valueBuffer();
+	auto* amountBuffer = m_amountModel.valueBuffer();
 	int amountInc = amountBuffer ? 1 : 0;
-	float *amountPtr = amountBuffer ? &(amountBuffer->values()[ 0 ] ) : &amount;
+	float* amountPtr = amountBuffer ? &(amountBuffer->data()[0]) : &amount;
```

Comment:
You can set the `amount` variable up top to `m_amountModel.valueAt(0)`.
```suggestion
```

Reply:
done